app/src/main/java/com/arctouch/codechallenge/util/BindAdapters.kt

```diff
@@ -0,0 +1,23 @@
+package com.arctouch.codechallenge.util
+
+import android.view.View
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import com.arctouch.codechallenge.R
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+
+@BindingAdapter("android:visibility")
+fun setVisibility(view: View, isVisible: Boolean) {
+    view.visibility = if (isVisible) View.VISIBLE else View.GONE
```

Comment:
Hello, i choose GONE exactly because i need to this view don't appear INVISIBLE in the layout, if  i use INVISIBLE, the view will be there occupying the space and invisible.